cve_bin_tool/format_checkers.py

```diff
@@ -76,9 +78,37 @@ def update_checker_table(file_path, markdown):
         f.writelines(lines)
 
 
+def update_allowed_words(checkers_array):
+    """Updates the allow.txt file with the new checkers"""
+    file_path = os.path.join(
+        os.path.abspath("."), ".github", "actions", "spelling", "allow.txt"
+    )
+
+    checkers_words = re.findall(r"[^0-9_]+", "_".join(checkers_array))
+    fileObj = open(file_path, "r")
+    words = fileObj.read().splitlines()
+    fileObj.close()
+
+    dict = requests.get(
+        "https://raw.githubusercontent.com/check-spelling/check-spelling/dictionary/dict.txt"
+    ).text.splitlines()
+
+    for checker in checkers_words:
+        if checker not in words and checker not in dict:
+            words.append(checker)
+
+    words = sorted(words, key=lambda v: v.upper())
```

Comment:
```suggestion
    words = sorted(words, key=str.casefold)
```
You can use [casefold](https://docs.python.org/3/library/stdtypes.html#str.casefold) which is designed for caseless sorting.